xml/deployment_sysreqs.xml

```diff
@@ -202,9 +202,7 @@
      </para>
      <para>
       The pre-installed VM images come with the relevant hypervisor's guest
-      tools pre-installed (where this is applicable). It is possible to install
-      these manually, but due to the read-only root filesystem, this is not a
-      trivial task.
+      tools pre-installed (where this is applicable).
```

Comment:
Pre-installed images with pre-installed tools? How about simply "come with the relevant hypervisor's guest tools?"

Reply:
Done.

Reply:
I've removed the redundant text. I have also added a single line with a short list of available virtual disk image formats.

The "second part of the bug"? Needing more memory on VMware? This is already addressed in the very next paragraph, on disabling the VMware memory-ballooning feature.